### PR TITLE
Re-enabled py2k build

### PR DESCRIPTION
**Pull request by kpalin**

### CODE REVIEW

**Comment by ericmjl** [comment]
ping @ocefpaf @jakirkham first time I'm seeing an error like this with the Linter - the py3k-only selector line has been deleted, but the linter seems to be failing on that exact non-existent line, I think.

**Comment by ericmjl** [comment]
Oh shucks, I'm a sloppy code reader... @kpalin, please see line-by-line comments, one other fix is needed.

File: recipe/meta.yaml [hunk 1]
```diff
@@ -13,7 +13,6 @@ source:
 
 build:
   number: 0
-  skip: True  # [py2k]
   script: python setup.py install --single-version-externally-managed --record record.txt
 
 requirements:
```
Comment (ericmjl) [inline recipe/meta.yaml]: Not this exact line, but on line 35, could you change:

` - enum34 >=1.1.6 # [py<34]`

to:

`- enum34 >=1.1.6  # [py<34]`

?

Note - there's just one extra space in there.

Comment (ocefpaf) [reply recipe/meta.yaml]: There is a PR in conda-smithy that will output the offending line to make situations like these easier to fix in the near future.

Comment (ericmjl) [reply recipe/meta.yaml]: Thanks @ocefpaf!

Comment (ericmjl) [reply recipe/meta.yaml]: Ping @kpalin, please see comment above re: `enum34`.

Comment (kpalin) [reply recipe/meta.yaml]: Sorry. I'll plea "inexperience with 'git commit' syntax" It's coming now.

**Comment by ericmjl** [comment]
No worries! Looks like everything is passing just fine.

**Comment by twiecki** [comment]
Thanks!

**Pull request merged**